### DISCUSSION TITLE
Inspired Color Gamut and Accessibility Visualization/Communication

### DISCUSSION THREAD

**Discussion by cgrobb** [meodai/color-names#140] [General]
https://oklch.evilmartians.io/
https://slides.com/ai/oklch-css#/42
https://huetone.ardov.me/

Were all new to me. Great color-related design and dev.

**Comment by meodai** [comment]
@cgrobb I wonder how https://oklch.evilmartians.io/ was made so fast! I struggle doing something similar

https://user-images.githubusercontent.com/608386/205310212-627cccfe-0fb6-403e-aa0e-5a7cfc839f5a.mov
